USING.md

```diff
@@ -2,23 +2,41 @@
+Asking and reporting in LevelSpace is conceptually pretty straight forward: You pass blocks of code to child models, and the child models respond as if you had typed that code into their Command Center. LevelSpace allows you to report strings, numbers, and lists from a child to its parent. It is not possible to directly report turtles, patches, links, or any of their respective sets. Further, it is not possible to push data from a child to its parent - parents must ask their children to report. This mimicks the way in which turtles cannot "push" data to the observer, but rely on the observer to ask them for it.
 
-LevelSpace has two different child model types, headless models and interactive models. They each have their strengths and weaknesses: Headless models are faster and use less memory than interactive models. Interactive models allow you full access to their interface and widgets, whereas headless models only give you access to their view and command center. Typically you will want to use headless models when you are running a large number of models, or if you simply want to run them faster - interactive models are good if you run a small amount of models, or if you are writing a LevelSpace model and need to be able to debug.
+In general, the LevelSpace syntax has been designed to align with existing NetLogo primitives whenever possible.
+
+### Headless and Interactive Models 
+
+LevelSpace has two different child model types; headless models and interactive models. They each have their strengths and weaknesses: 
+
+Interactive models 
+* are full-fledged models that give full access to their interface and widgets,
+* run a bit slower, and use more memory
+
+Headless Models
+* only give you access to their view and command center 
+* are faster and use less memory than interactive models. 
+
+Typically you will want to use headless models when you are running a large number of models, or if you simply want to run them faster. Interactive models are good if you run a small amount of models, if you are writing a LevelSpace model and need to be able to debug, or if you need access to widgets during runtime.
+
+### Keeping Track of Models
 
 Child models are kept track of in the extension with an id number, starting with 0, and all communication from parent to child is done by referencing this number, henceforth referred to as `model-id`.
 
 The easiest way to work with multiple models is to store their `model-id` in a list, and use NetLogo's list primitives to sort, filter, etc. them during runtime.
 
-### A general usecase
+Keeping track of models is important: Most LevelSpace primitives will fail and cause a runtime interruption if provided a `model-id` to a non-existing model. 
```

Comment:
Absolutely, especially for novices. In fact, now that I think about it, we should add a usecase for updating lists to keep them up to date :p